libs/core/src/lib/select/select.component.html

```diff
@@ -9,8 +9,7 @@
             <ng-container *ngTemplateOutlet="triggerTemplate; context: {$implicit: this}"></ng-container>
         </ng-container>
         <ng-container *ngIf="!triggerTemplate">
-            <button class="fd-select-button-custom"
```

Comment:
There let's remove `[compact]` and add `fd-button--compact: compact` to `[ngClass]`.

Reply:
@JKMarkowski currently its in the same format as u defined.
<img width="489" alt="Screenshot 2019-12-02 at 9 21 44 PM" src="https://user-images.githubusercontent.com/53534379/69973589-c5969f00-1549-11ea-83c4-79c8ba4451e4.png">